pkg/apis/core/v1beta1/types.go

```diff
@@ -152,14 +152,33 @@ type DataVolumeSourceS3 struct {
 
 // DataVolumeSourceRegistry provides the parameters to create a Data Volume from an registry source
 type DataVolumeSourceRegistry struct {
-	//URL is the url of the Docker registry source
+	//URL is the url of the registry source (starting with docker:// for Docker, or is:// for ImageStream)
 	URL string `json:"url"`
+	//ImportMethod is the import method: "skopeo" (default), or "cri"
+	ImportMethod RegistryImportMethod `json:"importMethod,omitempty"`
 	//SecretRef provides the secret reference needed to access the Registry source
 	SecretRef string `json:"secretRef,omitempty"`
 	//CertConfigMap provides a reference to the Registry certs
 	CertConfigMap string `json:"certConfigMap,omitempty"`
 }
 
+const (
+	// RegistryTransportDocker is docker transport prefix
+	RegistryTransportDocker = "docker"
```

Comment:
```go
// RegistrySchemeDocker is docker scheme prefix
RegistrySchemeDocker = "docker"
// RegistrySchemeOci is oci-archive scheme prefix
RegistrySchemeOci = "oci-archive"
// RegistrySchemeImageStream is image stream scheme prefix
RegistrySchemeImageStream = "is"
```
Please also fix https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/importer/transport.go#L88-L92 to use these constants. Could be a follow up PR.

Reply:
Fixed